ec2net-functions

```diff
@@ -161,6 +161,14 @@ get_secondary_ipv4s() {
   return $ec
 }
 
+get_delegated_prefix() {
+    local af=$1
+    local prefixes=($(get_meta "${af}-prefix"))
+    ec=$!
```

Comment:
I assume you meant `$?` instead of `$!`. Also, the `local` command will mask the return value such that the command always appears to succeed.

```suggestion
    local prefixes
    prefixes=($(get_meta "${af}-prefix"))
    local ec=$?
```

Reply:
`$!` vs. `$?` is correct, thanks for catching this.

The `prefixes` variable should be local in scope. It is communicated to the caller on stdout via the echo call on line 168.